src/test/kotlin/dsl/ResumeTest.kt

```diff
@@ -0,0 +1,37 @@
+package dsl
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class ResumeTest:StringSpec({
+
+    "이력서 dsl 생성 테스트" {
+        val result = introduce {
+            name("박재성")
+            company("우아한형제들")
+            skills {
+                soft("A passion for problem solving")
+                soft("Good communication skills")
+                hard("Kotlin")
+            }
```

Comment:
p3; 아래와 같이 의도치 않은 dsl 구문을 생성하더라도 제어할 수 있는 힘이 없을 거 같아요.
DslMarker에 대해 찾아보면 해답을 찾을 수 있습니다. 😊

```suggestion
            skills {
                languages {                     // 기술 내부 언어에 대한 정보
                    "Korean" level 5
                    "English" level 3
                }
                soft("A passion for problem solving")
                soft("Good communication skills")
                hard("Kotlin")
            }
```